.github/workflows/e2e-app-store.yml

```diff
@@ -39,4 +39,4 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: app-store-results-${{ matrix.shard }}_${{ strategy.job-total }}
-          path: app-store-results
+          path: test-results
```

Comment:
app-store-results isn't the correct path. CI isn't showing the artifacts for embed and app-store tests right now.